.gitlab-ci.yml

```diff
@@ -15,5 +15,5 @@ libtelio-build-pipeline:
 
   trigger:
     project: $LIBTELIO_BUILD_PROJECT_PATH
-    branch: v0.4.13
+    branch: LLT4813_lock_python_deps
```

Comment:
Adding a message to serve as a reminder to update this to semver once build automation changes is merged